node/Dockerfile.s390x

```diff
@@ -12,23 +12,27 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-ARG QEMU_IMAGE=calico/go-build:latest
-ARG BIRD_IMAGE=calico/bird:latest
+ARG BIRD_IMAGE
 
-FROM ${QEMU_IMAGE} as qemu
-FROM ${BIRD_IMAGE} as bird
+FROM alpine:3.11 as qemu
```

Comment:
I'm still not sure why we need to change from QEMU_IMAGE -> alpine:3.11 here. 

Our build process is designed around a common set of arguments passed to each build call:
- QEMU_IMAGE
- BIRD_IMAGE
- UBI_IMAGE

We should use the values that are passed in rather than defining something else within the Dockerfile, unless there's a really solid reason not to (but I'm not yet aware of one in this case)

Reply:
> I'm still not sure why we need to change from QEMU_IMAGE -> alpine:3.11 here.
> 
> Our build process is designed around a common set of arguments passed to each build call:
> 
>     * QEMU_IMAGE
> 
>     * BIRD_IMAGE
> 
>     * UBI_IMAGE
> 
> 
> We should use the values that are passed in rather than defining something else within the Dockerfile, unless there's a really solid reason not to (but I'm not yet aware of one in this case)

@caseydavenport this is because for s390x, we can not use `COPY --from=qemu /usr/bin/qemu-${ARCH}-static /usr/bin/` to use the `qemu-${ARCH}-static`, we'll need use the version `6.1.0-8`. Which is downloaded in `RUN curl -L https://github.com/multiarch/qemu-user-static/releases/download/v${QEMU_VERSION}/qemu-${QEMU_ARCHS}-static.tar.gz | tar zxvf - -C /usr/bin &&\`, so QEMU_IMAGE in this case is just used to run the `curl` command.

Reply:
> I'm still not sure why we need to change from QEMU_IMAGE -> alpine:3.11 here.
> 
> Our build process is designed around a common set of arguments passed to each build call:
> 
>     * QEMU_IMAGE
> 
>     * BIRD_IMAGE
> 
>     * UBI_IMAGE
> 
> 
> We should use the values that are passed in rather than defining something else within the Dockerfile, unless there's a really solid reason not to (but I'm not yet aware of one in this case)

Looks the qemu-s390x-static also works in QEMU_IMAGE but don't need specify the version, should the new go-build image included it already, I tried it in apiserver, so test others. I knew how to run the CI any way.

Basic the ci is running:
```
make generate
make check-dirty
cd $COMPONENT
make ci
```